src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -143,7 +148,7 @@ def login(cmd, username=None, password=None, service_principal=None, tenant=None
         logger.warning(_CLOUD_CONSOLE_LOGIN_WARNING)
 
     if username:
-        if not (password or client_assertion):
```

Comment:
Shall we also add certificate check in https://github.com/Azure/azure-cli/blob/6c32c4de828570e98053c270ede03479fe521153/src/azure-cli/azure/cli/command_modules/profile/custom.py#L123-L126